src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,44 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+interface Props {
+  todo: Todo | null;
+  setModalOpen: (val: boolean) => void;
```

Comment:
The `setModalOpen` function is expected to take a boolean value, but in the `handleCloseModal` function, it is called with `false`. Ensure that this aligns with the intended functionality, as the function signature suggests it should be used to open or close the modal.